spyder/plugins/explorer/widgets.py

```diff
@@ -851,7 +877,181 @@ def create_func(fname):
+                                file_no_ext, file_ext = osp.splitext(base_name)
+                                end_number = re.search(r'\d+$', file_no_ext)
+                                if end_number:
+                                    new_number = int(end_number.group()) + 1
+                                else:
+                                    new_number = 1
+                                left_string = re.sub(r'\d+$', '', file_no_ext)
+                                left_string += str(new_number)
+                                base_name = left_string + file_ext
+                                destination = osp.join(parrent_path, base_name)
+                            else:
+                                destination = osp.join(parrent_path, base_name)
+                            shutil.copy(source_name, destination)
+                        except Exception as e:
+                            QMessageBox.critical(self, _('Error Pasting File'),
+                                                 _("Unsupported Copy "
+                                                   "Operation:\n\n")
+                                                 + to_text_string(e))
+                    else:
+                        try:
+                            while base_name in os.listdir(parrent_path):
+                                end_number = re.search(r'\d+$', base_name)
+                                if end_number:
+                                    new_number = int(end_number.group()) + 1
+                                else:
+                                    new_number = 1
+                                left_string = re.sub(r'\d+$', '', base_name)
+                                base_name = left_string + str(new_number)
+                                destination = osp.join(parrent_path, base_name)
+                            else:
+                                destination = osp.join(parrent_path, base_name)
+                            if osp.realpath(destination).startswith(
+                                    osp.realpath(source_name) + os.sep):
+                                QMessageBox.critical(self,
+                                                     _('Recursive Copy'),
+                                                     _("Source is an ancestor"
+                                                       " of destination"
+                                                       " folder."))
+                                continue
+                            shutil.copytree(source_name, destination)
+                        except Exception as e:
+                            QMessageBox.critical(self,
+                                                 _('Error Pasting Folder'),
+                                                 _("Unsupported Copy"
+                                                   " Operation:"
+                                                   "\n\n") + to_text_string(e))

```

Comment:
Let's change this to

```python
_("Unsupported copy "
  "operation. The error was:\n\n")
  + to_text_string(e)
```

taking into account not to going beyond 80 columns, of course.